src/main/kotlin/uk/gov/gdx/datashare/service/DataProcessor.kt

```diff
@@ -23,37 +24,23 @@ class DataProcessor(
     val dataProcessorMessage: DataProcessorMessage = objectMapper.readValue(message, DataProcessorMessage::class.java)
     log.info("Received event [{}] from [{}]", dataProcessorMessage.eventTypeId, dataProcessorMessage.publisher)
 
-    // lookup provider
-    val details = getDataFromProvider(dataProcessorMessage)
+    val dataId = dataProcessorMessage.id ?: UUID.randomUUID().toString()
 
-    when (dataProcessorMessage.eventTypeId) {
-      "DEATH_NOTIFICATION" -> deathNotificationService.saveDeathNotificationEvents(
-        details,
-        dataProcessorMessage,
-      )
-
-      "LIFE_EVENT" -> print("x == 2")
+    if (dataProcessorMessage.datasetId != "DEATH_LEV" && dataProcessorMessage.datasetId != "PASS_THROUGH") {
```

Comment:
Can we have passthrough?

Reply:
It was in the codebase beforehand so I moved it over to here, and seems to be in our db, but I'm not sure exactly what it's used for at this point